client/src/components/child-components/MemoryTile.js

```diff
@@ -43,10 +43,11 @@ const ViewComments = (props) => {
     const m = props.m;
     const submitComment = props.submitComment;
     const comments = props.comments;
-    const getComments = props.getComments;
```

Comment:
So the best practice here is to use destructuring:

const {m, submitComment, comments, getComments} = props